content/_index.md

```diff
@@ -16,6 +16,7 @@ The guides section has an overview of major concepts in IPFS, guides, and exampl
 
 In particular, IPFS is a complex system that aims to change a lot of things about how we use the internet, so it naturally comes with a lot of new concepts. Check the [concepts](/guides/concepts) section to learn more about the major architectural pieces of IPFS and about terms and ideas associated with distributed file systems.
 
+If you prefer “hands-on” approach, [ProtoSchool](https://proto.school) will introduce you to decentralized web by writing code and solving challenges.
```

Comment:
Love the idea of adding links! Suggestion for a slight wording tweak here:

"If you prefer a hands-on approach, try out the interactive tutorials at [ProtoSchool](https://proto.school), where you can learn about the decentralized web by solving code challenges."

Reply:
Thanks, changed in: https://github.com/ipfs/docs/pull/146/commits/ab15273065956175856e34112cb6e5f594337410